Source/Analytics/Web/src/components/Reports/ReportsPerHealthRiskPerDay.js

```diff
@@ -84,25 +84,42 @@ export default class ReportsPerHealthRiskPerDay extends Component {
     renderRegions() {
         return this.state.regions.map(region => (
             <TableRow key={region.name}>
-                <TableCell className="headerText" className="cell">{region.name}</TableCell>
+                <TableCell className="headerText" className="cell" align="center">{region.name}</TableCell>
                 {this.renderReports(region.name)}
             </TableRow>)
         );
     }
 
     renderDays() {
         return this.state.reportsPerHealthRisk.map(report => {
-            const day = new Date(report.timestamp).toDateString();
-            return <TableCell className="headerText" key={report.id}>{day}</TableCell>
+            const day = new Date(report.timestamp);
+            return <TableCell className="headerText date" align="center" key={report.id}>
```

Comment:
Only show the date of this format "28.07" (not today and not the word reports)

Reply:
I spoke to Hannah and we agreed on this format. But I can update it to this instead.